isodatetime/data.py

```diff
@@ -1750,11 +1746,11 @@ def _get_days_in_year_range(start_year, end_year, calendar_mode=None):
 
 def get_days_in_year(year):
     """Return the number of days in this particular year."""
-    return _get_days_in_year(year, calendar_mode=CALENDAR.mode)
+    return _get_days_in_year(year, CALENDAR.mode)
 
 
 @util.cache_results
-def _get_days_in_year(year, calendar_mode=None):
+def _get_days_in_year(year, _):
```

Comment:
If the argument isn't used, why does it need a 'discardable' argument passing? (I must be missing something here...)